instrumentation/aws-lambda/aws-lambda-core-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awslambdacore/v1_0/AwsLambdaRequestHandlerInstrumentation.java

```diff
@@ -59,6 +60,8 @@ public void transform(TypeTransformer transformer) {
   @SuppressWarnings("unused")
   public static class HandleRequestAdvice {
 
+    private static final long flushTimeoutNanos = flushTimeout().toNanos();
```

Comment:
We don't add fields to advice classes. Either inline it or move the field to https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/aws-lambda/aws-lambda-core-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awslambdacore/v1_0/AwsLambdaInstrumentationHelper.java
It would be best to keep changes here aligned with https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/4497fbf968c20becdcc998e8bacc849e2eeafc21/instrumentation/aws-lambda/aws-lambda-events-2.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awslambdaevents/v2_2/AwsLambdaRequestHandlerInstrumentation.java#L109